examples/isostasy_calculation.py

```diff
@@ -0,0 +1,54 @@
+"""
+Calculation of
```

Comment:
The example script needs to start with a title that will be used as the title for the plot in the gallery. For this one, `Airy Isostasy` should be good.

Reply:
After, we just need a short paragraph about what you're showing in this example.